CMakeLists.txt

```diff
@@ -1330,7 +1330,7 @@ target_link_libraries(mixxx-lib PUBLIC
   Qt5::Test
   Qt5::Widgets
   Qt5::Xml)
-target_compile_definitions(mixxx-lib PUBLIC QT_TABLET_SUPPORT)
+target_compile_definitions(mixxx-lib PUBLIC QT_TABLET_SUPPORT QT_USE_QSTRINGBUILDER)
```

Comment:
I am unsure about that.

With QT_USE_QSTRINGBUILDER defined, operator + uses the stringbuilder as well.

This means your manual changes are redundant. 

It might be surprising for new contributors.

Has the stringbuilder any benefit if we only concatenate two strings?

Reply:
I replaced + with % to indicate usage of QStringBuilde where I had to fix the type conversions. Both operators map to QStringBuilder with QT_USE_QSTRINGBUILDER, so it doesn't matter.

Reply:
Distinguishing between 2 and more strings complicates the code. Let's keep it simple and use what is optimized for the general case so that developers don't need to care.

Reply:
Ah, I have now understand the PR.
You have "just" set the QT_USE_QSTRINGBUILDER macro and fixed the build.
I originally thought you did a search for string concatenations as well.

I am unsure what is the best approach now for readability. I did not meant to request to replace % with +. On one hand + is better to read and use, on the other hand % matches the Qt docu.

Do you think we should user + everywhere in future?

Reply:
+ is more common, and even more with copy elision of return values in modern C++. The % is just a workaround that was needed, because the QStringBuilder optimization is not backward compatible as we see here.

Reply:
So let's simply use + and don't confuse developers.

Reply:
Type incompatibilities can always be solved by assigning to a temporary QString.